manage.py

```diff
@@ -2,23 +2,14 @@
 import os
 import sys
 
-if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "pyfoss.settings")
-
+if __name__ == '__main__':
```

Comment:
```manage.py``` file is generated by django.
This should not be modified.
You should create a new project using django 2.1 and clone the source code only.

Reply:
Git showing the difference only. It is the default file created by django 2.1.7